src/user-icon/user-icon.scss

```diff
@@ -136,60 +136,91 @@
       color: t(iui-text-color-muted);
     }
   }
-}
-
-@mixin iui-user-icon-list {
-  display: flex;
 
-  .iui-user-icon:not(:first-child) {
-    margin-left: -$iui-xs;
+  &.iui-user-icon-count:focus {
+    @include themed {
+      box-shadow: t(iui-focus-box-shadow);}
+    border-radius: 50%;
 
-    .iui-initials {
+    &.iui-user-icon > .iui-stroke {
       @include themed {
```

Comment:
`&.iui-user-icon` is pretty redundant. Is it intentional just to increase specificity?

Reply:
removed iui-user-icon